NGitLab/Impl/GroupsClient.cs

```diff
@@ -112,6 +117,18 @@ public Task<Group> GetByFullPathAsync(string fullPath, CancellationToken cancell
             return _api.Get().ToAsync<Group>(Url + "/" + Uri.EscapeDataString(fullPath), cancellationToken);
         }
 
+        public GitLabCollectionResponse<Group> GetSubgroupsByIdAsync(int id, GroupQuery query)
```

Comment:
I think query should allow null and be optional. Something like 

```suggestion
        public GitLabCollectionResponse<Group> GetSubgroupsByIdAsync(int id, GroupQuery query = null)
```

Reply:
I agree I have made the changes.